### ISSUE TITLE
Swiping from the center of the screen causes previous tabs to become visible

### ISSUE THREAD

**Issue by WesselKuipers** [Awful/Awful.apk#360]
When swiping to the left or right from the middle causes the tabs to the left or right to gradually nudge back on the screen the more you keep swiping.

This can become a bit of an annoyance when you're quickly trying to open the side menu, as well as blocking out parts of the screen and generally not being pretty to look at.

![image](https://cloud.githubusercontent.com/assets/4731930/10113078/1cb06be6-63e1-11e5-8ee2-19dbaa4597fb.png)

**Comment by baka-kaba** [comment]
Is it stuck on that screenshot? I know there's an issue with swiping in code blocks and the like, but is it sticking on the forum/thread lists too? Does it happen all the time, and is it the same with Page Transformations disabled in the Advanced settings? (You might have to restart the app to get that one to take)

**Comment by WesselKuipers** [comment]
Yeah it was stuck in that screenshot. You'd still be able to interact with both of the pages, which ends up looking kinda wonky.

Turns out though that when I posted about this issue, I wasn't running the most recent version. The version I was running at the time didn't have the 'smooth' page transformation effect.

In the older version it happened all the time when swiping repeatedly from the middle of the screen. Now it only seems to happen in code blocks like you already mentioned. Turning off Page Transformations in the settings doesn't seem to change anything.
